test/e2e/scheduler_predicates.go

```diff
@@ -125,6 +125,24 @@ func cleanupPods(c *client.Client, ns string) {
 	}
 }
 
+func removeLabelOffNode(c *client.Client, nodeName string, labelKey string) {
+	By("removing the label " + labelKey + " off the node " + nodeName)
+	framework.RunKubectlOrDie("label", "nodes", nodeName, labelKey+"-")
```

Comment:
Oops, this line should be removed.
The lines following are in the way ordinary update.